tilelang/autotuner/tuner.py

```
@@ -424,17 +437,416 @@ def _load_result_from_disk(self, key) -> AutotuneResult:
+                        _worker_state: _BenchmarkWorkerState = worker_state,
+                        _call_result_queue: queue.SimpleQueue = call_result_queue,
+                    ):
+                        try:
+                            latency, worker_ref_latency = benchmark_target(
+                                jit_kernel=_jit_kernel,
+                                benchmark_state=_worker_state,
+                            )
+                            _call_result_queue.put(("ok", latency, worker_ref_latency, ""))
+                        except TimeoutException:
+                            _call_result_queue.put(("timeout", None, None, ""))
+                        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)